insights/terraform/projects/bfd/api-requests/glue.tf

```diff
@@ -25,25 +21,1306 @@ module "glue-table-api-requests" {
 
   partitions = [
     {
-      name    = "year"
-      type    = "string"
-      comment = "Year of request"
+      name    = "year"
+      type    = "string"
+      comment = "Year of request"
+    },
+    {
+      name    = "month"
+      type    = "string"
+      comment = "Month of request"
+    }
+  ]
+
+  # We have to explicitly specify the schema here, because Firehose cannot export to a parquet
```

Comment:
Just curious here on how you derived the list of columns / names? via some sort of script?

Reply:
Trotter found them via Athena. But yeah, it was basically just a query that grabbed all of the columns from exported log files dating back to 2019.